admin/services/data-access/sql.service.js

```diff
@@ -476,17 +523,15 @@ sqlService.generateUpdateStatement = async (table, data) => {
  * Create a new record
  * @param {string} tableName
  * @param {object} data
+ * @param {boolean} dropRedisCaches
  * @return {Promise} - returns the number of rows modified (e.g. 1)
  */
-sqlService.create = async (tableName, data) => {
+sqlService.create = async (tableName, data, dropRedisCaches = false) => {
   const preparedData = convertMomentToJsDate(data)
-  const {
-    sql,
-    params,
-    outputParams
-  } = await sqlService.generateInsertStatement(tableName, preparedData)
+  const { sql, params } = await sqlService.generateInsertStatement(tableName, preparedData)
   try {
-    const res = await sqlService.modify(sql, params, outputParams)
+    const res = await sqlService.modify(sql, params, dropRedisCaches ? tableName : false)
```

Comment:
what's happened to the outputParams?

Reply:
`generateInsertStatement` did used to return `outputParams`, but there is only one place in the codebase using `generateInsertStatement` and it doesn’t use the `outputParams` value. So I removed it as it’s not being used.

Reply:
ok, thanks for clarification